Modules/Utils/Sources/PocketCastsUtils/Extensions/ComparableExtension.swift

```diff
@@ -9,3 +9,13 @@ public extension Comparable {
         self > limits.lowerBound && self < limits.upperBound ? self : self.clamped(to: limits)
     }
 }
+
+public extension Comparable {
+    func clamped(to limits: ClosedRange<Self>) -> Self {
```

Comment:
These are duplicates of the `Range` versions for `ClosedRange`. Unfortunately, they don't share a suitable protocol so I couldn't make these generic.